CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # go-ipfs changelog
 
+Ipfs 0.4.16 is a fairly small release in terms of changes to the ipfs codebase,
+but it contains a huge amount of changes and improvements from libraries we
+depend on, notably libp2p. This version of ipfs contains the changes for libp2p
+v6.0.5, In that release, we made significant changes to the codebase to allow
```

Comment:
Would it be helpful to say we upgraded from libp2p v5.0.14 to v6.0.5 so people know what range of changelog entries to look for if they want to dig into details on their own? (Also, do I have that version correct?)

Reply:
It says that further down but I've put that higher up in the version with paragraphs.